react/components/PickupDetailsModal.tsx

```diff
@@ -100,7 +105,13 @@ const PickupDetailsModal: React.VFC<Props> = ({
             <FormattedMessage id="store/checkout.shipping.pickupPointsModal.address" />
           </p>
 
-          <PlaceDetails />
+          <AddressContext.AddressContextProvider
+            address={pickupOption.address}
+            countries={countries}
+            rules={rules}
```

Comment:
I think `countries` and `rules` must be synced with the pickupOption.address location. Can we even have access to that information?

Reply:
what do you mean by "access to that information"? which information? I think this shouldn't be any different than what we have now, because the pickup address will be in the same country of the deliveries a particular store has, so if it was correctly configured with the rules for the country the formatation of the pickup address will work as usual

Reply:
Ah, I mean the country and rules information for the pickup option, as it can be from another country.

> because the pickup address will be in the same country of the deliveries a particular store has

My doubt was related to a store with multiple countries, but I just realized that this doesn't matter. `PlaceDetails` renders everything based on the address country anyway. Thanks!